### ISSUE TITLE
"Failed to collect package" results in clean exit with "No issues found"

### ISSUE THREAD

**Issue by bitttttten** [QuiiBz/sherif#103]
If we run sherif on our monorepo we see:

```sh
$ npx sherif
 ⨯ error Failed to collect package
   Error while parsing "./packages/c/package.json": invalid type: string "true", expected a boolean at line 4 column 19

✓ No issues found
```

Although on circleci this passes with a green ✅ , i.e. `npx sherif && echo 'hello'` will echo hello. Should it be that this error should also fail the linter and exit?

**Comment by bitttttten** [comment]
The problem with our package.json was that we had `"private": "true",` instead of `"private": true,`.

**Comment by QuiiBz** [comment]
Thanks for the report, that happens when we fail to parse a `package.json`. [I released version 1.0.2](https://github.com/QuiiBz/sherif/releases/tag/v1.0.2) that includes https://github.com/QuiiBz/sherif/pull/104
